src/mercury-agent/mercury/agent/agent.py

```diff
@@ -92,28 +92,48 @@ def run(self, dhcp_ip_method='simple'):
         agent_service.start()
 
 
-def _parse_args():
+def parse_args():
     parser = argparse.ArgumentParser(description='Mercury Agent')
+    parser.add_argument('-c', '--config-file', default=None, help='Path to agent configuration file')
+    parser.add_argument('-d', '--debug', default=False, action='store_true', help='Enable debug logging')
+    return parser.parse_args()
 
 
-def main():
+def setup_logging():
+    log_level = logging.getLevelName(agent_configuration['agent']['log_level'])
+    logging.basicConfig(level=log_level)
```

Comment:
Is this just for dubgging I imagine? having a stream handler on the root debugger? I imagine we want to move away from this when possible.

Reply:
That is so we seem messages on the console prior to setting up the remote logger